<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamemap3</class>
 <widget class="QWidget" name="gamemap3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelmap3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>861</width>
     <height>771</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Pictures/map3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Old English Text MT</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;MAP C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Stencil</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;SANHOK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WinGame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>650</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
   <property name="text">
    <string>NEXT!!!!!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleArthas">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(226, 107, 10);
}</string>
   </property>
   <property name="text">
    <string>阿尔
萨斯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleGilgamesh">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>140</y>
     <width>175</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(226, 107, 10);
}</string>
   </property>
   <property name="text">
    <string>吉尔伽美什</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleSteves">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>183</y>
     <width>175</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(192, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>三位史蒂夫</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleNo_99">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>300</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(192, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>No.99
霍普神龙</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattlePony">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>460</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(226, 107, 10);
}</string>
   </property>
   <property name="text">
    <string>谱尼</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleUSS_Enterprise">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>490</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(192, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>企业</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleMoon_Lord">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>660</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(192, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>月球
领主</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleEnder_Dragon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>511</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(0, 0, 0);
color: rgb(255,255, 255);
}</string>
   </property>
   <property name="text">
    <string>末影龙</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egaincard">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>311</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>法师的睿智：抽牌数永久加1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(112, 48, 160);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="EDagger">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>566</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>影之匕首：所有打出的威力大于等于24的单次攻击威力增加8</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(226, 107, 10);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egaindefense">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>464</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>圣物之盾：DEF永久加2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 176, 240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egaincost">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>362</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>魔力宝石：每回合获得MANA永久加2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(112, 48, 160);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egainlife">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>血脉项链：每场战斗结束后MAXHP永久加2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(112, 48, 160);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ERhoAias">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>515</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Rho Aias：所有受到的伤害大于等于20的单次攻击威力减少5</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(226, 107, 10);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egainattack">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>413</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>增幅宝珠：ATK永久加5</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 176, 240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
